lib/index.js

```diff
@@ -31,21 +31,21 @@ function Setup(T, { checkTypes , _env=[] }){
     }
 
     function mapConstrToFn(constr) {
-    return (
-        constr === String
-            ? T.String
-        : constr === Number
-            ? T.Number
-        : constr === Boolean
-            ? T.Boolean
-        : constr === Object
-            ? T.Object
-        : constr === Array
-            ? T.Array(T.Any)
-        : constr === Function
-            ? T.AnyFunction
-        : constr
-    )
+        return (
+            constr === String
+                ? T.String
+            : constr === Number
+                ? T.Number
+            : constr === Boolean
+                ? T.Boolean
+            : constr === Object
+                ? T.Object
+            : constr === Array
+                ? T.Array(T.Any)
+            : constr === Function
+                ? T.AnyFunction

```

Comment:
Yeah I get your argument.  I think though you can't clearly delineate "this type takes predicates, this type accepts types", often a user will want to do both.  Maybe we could provide a utility for easily creating anonymous predicate types, I'd be happy if those predicate's types' names were just the function body.  e.g. `$.NullaryType( 'x => x % 2 == 0', '', x => x % 2 == 0)` could just become `predicate( x => x % 2 )`.